packages/xo-server-perf-alert/src/index.js

```diff
@@ -373,8 +392,9 @@ ${monitorBodies.join('\n')}`
   }
 
   _parseDefinition(definition) {
-    const alarmId = `${definition.objectType}|${definition.variableName}|${definition.alarmTriggerLevel}`
-    const typeFunction = TYPE_FUNCTION_MAP[definition.objectType][definition.variableName]
+    const objectType = definition.objectType.toLowerCase()
```

Comment:
Make it explicit that this `objectType` is lowercase (you can name it `lcObjectType` if you want).

Otherwise, someone might replace `definition.objectType` with it in the code below.